File: quests/fearplane/a_broken_golem.pl
sub EVENT_ITEM {
	if($itemcount{14319} == 1){
		quest::say("Error! Malfunction! Destroy!");
		quest::say("A $class like you always bring out the worst in me.");
		quest::spawn2(72106,0,0,$x,$y,$z,$h);
		quest::depop();
	}
}

sub EVENT_SIGNAL {
 if($signal == 0) {
  quest::say("Such is the will of Cazic-Thule!");
 }
}

#Original by: ? Additions by: Jim Mills

File: fearplane/a_broken_golem.lua
-- items: 14319
function event_trade(e)
	local item_lib = require("items");
	
	if(item_lib.check_turn_in(e.trade, {item1 = 14319})) then
		e.self:Say("Error! Malfunction! Destroy!");
		e.self:Say("A " .. e.other:Class() .. " like you always bring out the worst in me.");
		eq.spawn2(72106,0,0,e.self:GetX(),e.self:GetY(),e.self:GetZ(),e.self:GetHeading()); -- NPC: an_enraged_golem
		eq.depop_with_timer();
	end

	item_lib.return_items(e.self, e.other, e.trade);
end
